plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncNuageVspCmsIdCommand.java

```diff
@@ -17,38 +17,33 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
+package com.cloud.agent.api.sync;
 
 import com.cloud.agent.api.Command;
 
-public class ReleaseVmVspCommand extends Command {
+public class SyncNuageVspCmsIdCommand extends Command {
```

Comment:
Per Java best practice, this class should implement the equals(Object), hashCode(), and toString() methods.